### DISCUSSION TITLE
Panning scaled and zoomed images with aspect ratios different from display to show hidden edges

### DISCUSSION THREAD

**Discussion by nameauser** [Geektoolkit/Dynaframe3#183] [Ideas]
It is great that images with aspect ratios different from the display can be zoomed and scaled. It would be amazing if this could be taken a step further by allowing panning so that the truncated areas could be shown by panning the whole image from one otherwise hidden edge to the other. I hope my description makes sense. I am pretty sure there is existing jargon for this

**Comment by Geektoolkit** [comment]
Yep! That's called the "Ken Burns Effect".  It's a great suggestion if someone wants to pick it up. I'm likely going to do it as part of Dynaframe pro at some point, but I need to work through some other features first. But it'd be great for artwork that goes off screen, esp. if it tied in with facial recognition to try to do it in a smart way
